### PR TITLE
moved --include-code to --format list and renames --format to --files

### PR DESCRIPTION
**Pull request by dan-and**
Moved --include-code  into the --format list, 
Also renamed --format to --files as you requested it.

### CODE REVIEW

File: downloader.py [hunk 1]
```diff
@@ -31,13 +31,13 @@ def main(argv):
 	email = ''
 	password = ''
 	directory = 'packtpub_media'
-	formats = 'pdf,mobi,epub,jpg'
+	fileTypes = 'pdf,mobi,epub,jpg,code'
 	includeCode = False
```
Comment (artifexor) [inline downloader.py]: You can remove includeCode variable as using other approach for downloading code.

Comment (dan-and) [inline downloader.py]: I didn't add "code" to the default list, as  "--include-code" is also not activated as default.

**Comment by dan-and** [comment]
@artifexor : Done

**Comment by ozzieperez** [comment]
@dan-and I'm closing this one out as these changes were merged into your previous one. 👍

**Pull request closed without merging**